client/components/data/moderate-comment/index.jsx

```diff
@@ -27,8 +27,7 @@ class ModerateComment extends Component {
 		this.moderate( this.props );
 	}
 
-	// @TODO: Please update https://github.com/Automattic/wp-calypso/issues/58453 if you are refactoring away from UNSAFE_* lifecycle methods!
-	UNSAFE_componentWillReceiveProps( nextProps ) {
+	componentDidUpdate( nextProps ) {
```

Comment:
It should be `prevProps` in this case, no?

Reply:
Ha, of course. Fixed in b2f59b8a56c729f9dd70d3c466ec2283cd12aff9